src/arkode/arkode_ls.c

```diff
@@ -2743,9 +2797,9 @@ int arkLsSolve(void* arkode_mem, N_Vector b, realtype tnow,
 #endif
 
   /* Interpret solver return value  */
-  arkls_mem->last_flag = retval;
+  arkls_mem->last_flag = ls_status;
 
-  switch(retval) {
+  switch(ls_status) {
 
   case SUNLS_SUCCESS:
     return(0);
```

Comment:
As above, should we change each of these return values to appropriate `SUNLsStatus` constants?